### PR TITLE
Add operating system to analytics

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

- Adds `operating_system` key to the payloads
- Adds the version data contexts to use the current versions in examples (because we can)

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/49725
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/analytics.markdown [hunk 1]
```diff
@@ -12,6 +12,8 @@ ha_codeowners:
 ha_domain: analytics
 ---
 
+{% assign current_version = site.current_major_version | append: "." | append: site.current_minor_version | append: "." | append: site.current_patch_version  %}
```
Comment (frenck) [inline source/_integrations/analytics.markdown:15]: Hehe nice detail :)

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/analytics.markdown [hunk 2]
```diff
@@ -74,12 +85,16 @@ If your system includes the Supervisor, this will also contain:
...
+  "version": "{{current_version}}",
   "installation_type": "Home Assistant OS",
   "supervisor": {
     "healthy": true,
     "supported": true
   },
+  "operating_system": {
+    "board": "odroid-n2",
+    "version:": "{{site.data.version_data.hassos['odroid-n2']}}"
+  }
```
Comment (balloob) [inline source/_integrations/analytics.markdown]:
```suggestion
  },
```

File: source/_integrations/analytics.markdown [hunk 3]
```diff
@@ -114,12 +129,16 @@ If your system includes the Supervisor, this will also contain:
...
```
Comment (balloob) [inline source/_integrations/analytics.markdown]:
```suggestion
  },
```

**Review by frenck — APPROVED** [review]

**Pull request merged**